### DISCUSSION TITLE
Compose Multiplatform

### DISCUSSION THREAD

**Discussion by ndvalkov** [peterLaurence/MapCompose#98] [General]
Hello,

Can I reuse MapCompose in a iOS app with Kotlin Multiplatform?

**Comment by p-lr** [comment]
Hello,

MapCompose isn't a KMM lib yet. While it's possible to do the required changes, I haven't myself tried it since my use case is for Android native development.
I would gladly accept contributions.

**Reply by ndvalkov** [reply]
It is a significant part of my app. I may try to adapt it, but not sure if I am capable.

**Reply by JakkeJ** [reply]
> It is a significant part of my app. I may try to adapt it, but not sure if I am capable.

Did you do it? I am in the same boat, but a lot of the stuff is Java based, like Inputstream, which I'm not sure is so easy to adapt to KMP/Compose Multiplatform
